src/plugins/formvalid/formvalid-en.hbs

```diff
@@ -425,7 +425,31 @@
 		&lt;/fieldset&gt;</code></pre>
 			</details>
 
-			<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset page to defaults" class="btn btn-link btn-sm show p-0 pt-4" />
+			<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset page to defaults" class="btn btn-link btn-sm show p-0 mrgn-tp-md" />
+		</form>
+	</div>
+</section>
+<hr>
+<section>
+	<h2>Example using custom pattern validation</h2>
+	<div class="wb-frmvld">
```

Comment:
Can you add a note like:

If the pattern attribute is used within the data-rule, the pattern validation will considerated before the data-rule. So at the end you get only it's generic error validation message.
It's recommended to use "data-msg" to customize the error message displayed by the pattern when the field is in error state.